rq_scheduler/utils.py

```diff
@@ -11,3 +13,9 @@ def from_unix(string):
 def to_unix(dt):
     """Converts a datetime object to unixtime"""
     return calendar.timegm(dt.utctimetuple())
+
+def crontab_get_next_scheduled_time(crontab_string):
```

Comment:
Can we rename the function signature to `get_next_scheduled_time(crontab_string)`?